compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -468,6 +469,10 @@ string t_java_generator::java_nullable_annotation() {
   return "@org.apache.thrift.annotation.Nullable";
 }
 
+string t_java_generator::java_override_annotation() {
+  return "@java.lang.Override";
```

Comment:
Everything in `java.lang` is imported by default. You don't need the fully-qualified class names for anything in there. But, I see it's done elsewhere (for String). I suppose it doesn't hurt.

```suggestion
  return "@Override";
```

Reply:
> it's done elsewhere (for String)

using fully qualified names (even for `java.lang.*`) allows for structs to be named `String` (although it's not a good practice) and used in the same package - otherwise it won't compile (some of the tests in `test/` folder does this).

For annotations i think it's fine to not to use fully qualified name.